letsencrypt/cli.py

```diff
@@ -139,9 +139,10 @@ def _determine_account(args, config):
         elif len(accounts) == 1:
             acc = accounts[0]
         else:  # no account registered yet
-            if args.email is None:
+            if args.email is None and not args.register_unsafely_without_email:
                 args.email = display_ops.get_email()
-            if not args.email:  # get_email might return ""
+            if not args.email:     # may be "" in integration tests?
```

Comment:
With the change I suggested to the integration test scripts, this `if` statement and its body can be removed.